netci.groovy

```diff
@@ -124,7 +124,7 @@ def CreateLinuxBuildTasks = { machine, configTag, linuxBranch, nonDefaultTaskSet
 
             def infoScript = 'bash jenkins/get_system_info.sh'
             def debugFlag = buildType == 'debug' ? '--debug' : ''
-            def buildScript = "bash ./build.sh -j=`nproc` ${debugFlag}"
+            def buildScript = "bash ./build.sh -j=`nproc` ${debugFlag} --cxx=/usr/bin/clang++-3.5 --cc=/usr/bin/clang-3.5"
```

Comment:
At the moment, 3.5 was all we had to work with. I'm talking to the owners to add clang 3.7